Trippy/View Controllers/Locate Tab/SearchMapViewController.m

```diff
@@ -163,4 +173,34 @@ - (void) addFetchedCollection:(LocationCollection *)collection {
     [self.collectionPickerView reloadAllComponents];
 }
 
+- (void) postedLocationSuccess:(Location *)location {
+    UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Success"
+                               message:@"Successfully posted location."
+                               preferredStyle:UIAlertControllerStyleAlert];
+    UIAlertAction* action = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
+                                   handler:^(UIAlertAction * action) {}];
```

Comment:
nit: not sure if the handler could be nil, but might be slightly nicer than having an empty block.